utils/build-script-impl

```diff
@@ -2487,6 +2487,11 @@ for host in "${ALL_HOSTS[@]}"; do
                   continue
                 ;;
                 *)
+                  # FIXME: Always re-build XCTest on non-darwin platforms.
+                  # Remove this when products build in the CMake system.
+                  echo "Cleaning the libdispatch build directory"
```

Comment:
Copy/paste error here? :-)

Reply:
hah, yeah, I'm not good at the copy paste thing